src/main/java/frc/robot/RobotContainer.java

```diff
@@ -189,6 +194,8 @@ private void configureBindings() {
         controller.x().onTrue(new LowerDeployer(deployer));
 //        controller.a().onTrue(new DeployerLower(deployer));
 //        controller.b().onTrue(new DeployerRaise(deployer));
+        AmpJoystick manualAmp = new AmpJoystick(amp, () -> controller.getLeftY());
```

Comment:
You can't use leftY, it's already mapped to the climber.  I also don't think we want this always mapped (this is temporary for testing) so you should wrap this in a DEBUG constant.